com.unity.netcode.adapter.utp/Runtime/UnityTransport.cs

```diff
@@ -517,13 +517,67 @@ private void Update()
                 {
                     ;
                 }
+                
+#if MULTIPLAYER_TOOLS
+                if (NetworkManager.Singleton.IsServer)
+                {
+                    var ngoConnectionIds = NetworkManager.Singleton.ConnectedClients.Keys;
+                    foreach (var ngoConnectionId in ngoConnectionIds)
+                    {
+                        ExtractNetworkMetrics(NetworkManager.Singleton.ClientIdToTransportId(ngoConnectionId));
+                    }
+                }
+                else
+                {
+                    ExtractNetworkMetrics(NetworkManager.Singleton.ClientIdToTransportId(NetworkManager.Singleton.ServerClientId));
+                }
+#endif
             }
         }
 
         private void OnDestroy()
         {
             DisposeDriver();
         }
+        
+#if MULTIPLAYER_TOOLS
+        private void ExtractNetworkMetrics(ulong transportClientId)
+        {
+            var networkConnection =  ParseClientId(transportClientId);
+            
+            //Two pipelines, unreliable sequence and reliable fragmented
```

Comment:
There are actually _3_ pipelines now. I added another one for unreliable (non-sequenced) delivery. You'll also need the change from PR #1569 for it to work properly with the metrics pipeline stage. (I guess you could just apply the changes to this PR and I'll close the other one.)

Reply:
I'll apply the change. It makes sense. Will fetch the last develop with the third pipeline.

Reply:
Comment needs updated (or maybe just removed?)

Reply:
I'd suggest having an enum for the pipeline stages and using the last member as a count. Is there an enum like that already around, I wonder? Migh be better documenting it that way than in comments.

Reply:
Yeah, I would just remove the comment. Better to have this kind of thing in an enum than a comment.